app/screens/home/index.tsx

```diff
@@ -133,17 +143,17 @@ export default function HomeScreen(props: HomeProps) {
                     <Tab.Screen
                         name={Screens.MENTIONS}
                         component={RecentMentions}
-                        options={{tabBarTestID: 'tab_bar.mentions.tab', lazy: true, unmountOnBlur: false, freezeOnBlur: true}}
+                        options={{tabBarTestID: 'tab_bar.mentions.tab', lazy: true, freezeOnBlur: true}}
                     />
                     <Tab.Screen
                         name={Screens.SAVED_MESSAGES}
                         component={SavedMessages}
-                        options={{unmountOnBlur: false, lazy: true, tabBarTestID: 'tab_bar.saved_messages.tab', freezeOnBlur: true}}
+                        options={{freezeOnBlur: true, lazy: true, tabBarTestID: 'tab_bar.saved_messages.tab'}}
```

Comment:
Nit: Just so it is visually more appealing... Can we reorder the other two options objects to have this same order? That way the part that is equal is visually equal in the three of them.